r_basics_introduction/r_basics_introduction.md

```diff
@@ -0,0 +1,784 @@
+
+This is what's called an **off-by-one indexing error** and is what happens when you use a tool like Excel and accidentally miss one row; or you have one dataset that has a header and another one that doesn't.  The result is that all the values in the affected column are shifted by one. This is a simple error to make, but it completely invalidates all downstream results.
+
+### Simple Errors
+
+The off-by-one indexing error was just one of many simple errors the MD Anderson team discovered.
+
+Another type of error that was pervasive in the study was label reversal so that cell lines were labeled sensitive to a drug when they actually were resistant, and vice versa. That type of error can lead to a scenario where a patient gets the chemotherapy that would be predicted to be least beneficial.
+
+Other problems they identified were confounding, inclusion of data from sources that were not reported in the paper, and wrong figure shown.
+
+These are all simple errors – people who make them aren't necessarily incompetent or negligent.
+
+Because these errors are so easy to make and because without good documentation or a reproducible workflow it's hard to catch them, they are also very common.
+
+A key issue in this case study is that the Duke investigators used "point and click" tools like Excel.
+
+This prevented peers and independent investigators from catching errors in the analysis, until it was too late.
+
+![Logo of Microsoft Excel](media/excel_logo.png)<!-- style = "max-width:200px" -->
+
+And the Duke case study is only one example where the barrier to reproducibility was that people used graphical user interface "point-and-click" type tools for analyzing large and complicated sets of data. Excel doesn't record user actions and because of this, is fundamentally not reproducible.
+
+### Future You
+
+Reproducibility doesn't only help people **outside** a study understand how things were done.  They also help the initial conduct of research as well.  Consider the following 3 statements and ask yourself if they sound familiar.
+
+- Can we redo the analysis with this month's data?
+- Why do the data in Table 1 not seem to agree with Figure 2?
+- Why did I decide to omit these six samples from my analysis?
+
+**Your closest collaborator is you from 6 months ago...**
+
+...  but they don't answer your emails!
+
+Additionally, we've already seen how irreproducibility can cause real harm to human subjects, something we all want to avoid.  
+
+In short, reproducible research can reduce harm for subjects, science, and scientists.  There are tools to help you conduct research data analysis in a way that helps your future collaborators, including future you, understand and extend your work.  A great tool for reproducible research is the use of R Markdown to record your research steps.
+
```

Comment:
I think it's helpful reinforcement, and it tees up why R Markdown matters.  The "reproducibility" module doesn't get down to use cases in this level of specificity, so I think this wouldn't be obvious there, either.  I'll work on some transitional text to smooth out the seams, though!

Reply:
That's fair, my concern is just "scope creep" -- the stated learning objectives of this module are all about getting someone familiar and comfortable with R, RStudio, and R Markdown, not convincing them of the importance of why things like literate statistical programming are important. If we want these modules to be truly "just in time" it might make more sense to have motivational content like this in a separate module like "Why X matters for the unconvinced". 

I also think this content could be super valuable for all sorts of folks, including those who only want to learn Python, e.g. 
While we know that folks in our pilot will all get this content, once the content is treated as intended (broadly disseminated a la carte usage as needed by a variety of research folks), this content will only be reaching people who want to learn R/RStudio/R Markdown and are beginning their education at the novice level. 

That being said, of course the editorial decision lies with you, and I don't think it would be terrible to leave it here!  -- just wanted to make my position a bit clearer. :)